docs/reference/cli/database.md

```diff
@@ -0,0 +1,725 @@
+
+Examples:
+```
+Request: "04:20:00:12:23:45 libraries getid<LF>"
+Request: "04:20:00:12:23:45 libraries getid id:8c7ee510 name:FLAC%20files%20only<LF>"
+```
+
+***
+## info total genres
+`info total genres ?`
+
+The `info total genres ?` query returns the number of unique genres in the server music database.
+
+Examples:
+```
+Request: "info total genres ?<LF>"
+Response: "info total genres 18<LF>"
+```
+
+***
+## info total artists
+`info total artists ?`
+
+The `info total artists ?` query returns the number of unique artists in the server music database. The `Composer, band and orchestra in artists` preference (Server Settings, Behavior) determines which contributors are considered artists.
+
+Examples:
+```
+Request: "info total artists ?<LF>"
+Response: "info total artists 18<LF>"
+```
+***
+## info total albums
+`info total albums ?`
+
+The `info total albums ?` query returns the number of unique albums in the server music database.
+
+Examples:
+```
+Request: "info total albums ?<LF>"
+Response: "info total albums 18<LF>"
+```
+
+***
+## info total songs
+`info total songs ?`
+
+The `info total songs ?` query returns the number of unique songs in the server music database.
+
+Examples:
+```
+Request: "info total songs ?<LF>"
+Response: "info total songs 18<LF>"
+```
+
+***
+## info total duration
+`info total duration ?`
+
+The `info total duration ?` query returns the number of seconds playtime in the server music database.
+
+Examples:
+```
+Request: "info total duration ?<LF>"
+Response: "info total duration 66109<LF>"
+```
+
+***
+## genres
+`genres <start> <itemsPerResponse> <taggedParameters>`
+
+The `genres` query returns all genres known by the server.
+
+Note that the server supports multiple genres per track, depending on the `Multiple items in tags` preference (Server Settings, Behavior).
+
+**Accepted tagged parameters:**
+
+| Tag | Description |
+|---|---|
+| `search` | Search string. The search is case insensitive and obeys the `Search Within Words` server parameter. |
+| `artist_id` | Limit results to those genres proposed by the artist identified by `artist_id`. |
+| `album_id` | Limit results to those genres available on the album identified by `album_id`. |
+| `track_id` | Limit results to the genres of the track identified by `track_id`. If present, other filters are ignored. |
+| `genre_id` | Limit results to the genre identified by `genre_id`. The genre_id may be a list of comma separated IDs. If present, other filters are ignored. |
+| `library_id` | Virtual library ID, to restrict the results to a virtual library view. |
+| `year` | Limit results to the genres of the tracks of the given `year`. |
+| `tags` | Determines which tags are returned. Each returned tag is identified by a letter (see below). The default value is empty. |
+
+**Returned tagged parameters:**
+
+| Block | Tag | Description |

```

Comment:
Same for this table, also hard to read

Reply:
Those tables could be better, agree fully - but not clear to me how to do this better.

Not making excuses, but this kind of complexity is not very easy to present within the extremely limited capabilities wrt Tables in mkdocs-material flavour Markdown. There is no "rowspan" or "colspan" directive, so cannot merge cells either vertically or horizontally. Neither can one put one table within the cells of another table.
Those capabilities are there in several other Markdowns I have worked with - for instance MediaWiki and Foswiki.

I have done some initial looks for additional modules for mkdocs-material, but could not see any which extend table functionality.

Reply:
You can always use raw HTML in such extreme cases.

Reply:
HTML wouldn't work. But there seems to be a hacky workaround: https://github.com/mkdocs/mkdocs/issues/1198#issuecomment-1253896100

```diff
diff --git a/docs/reference/cli/players.md b/docs/reference/cli/players.md
index 4791e83..c5ca3fe 100755
--- a/docs/reference/cli/players.md
+++ b/docs/reference/cli/players.md
@@ -581,7 +581,7 @@ Results returned as two blocks.
 || canpoweroff | Whether the player can be powered off. This value is false for streaming connections. Equivalent to `player canpoweroff <playerindex|playerid> ?` |
 || connected | Connected state. Equivalent to `<playerid> connected ?`. |
 || firmware | Firmware version |
-| For each defined pref requested:-|||
+| For each defined pref requested:- {: colspan=3} |&#8288 {: style="padding:0"}|&#8288 {: style="padding:0"}|
 || prefName | Preference value. Only if the value is defined. Equivalent to `<playerid> playerpref <prefname|namespace:prefname> <prefvalue|?>`. |
```

Reply:
The colspan-3 chagnes made to the relevant lines in the table in players.md